examples/sample-app/main.go

```diff
@@ -30,7 +35,13 @@ func main() {
 		if err != nil {
 			fmt.Printf("bad credentials: %s", err)
 		}
-		cfg := aws.NewConfig().WithEndpoint(endpointStr).WithRegion(regionStr).WithDisableSSL(true).WithCredentials(creds).WithS3ForcePathStyle(true)
+		cfg := aws.NewConfig().WithEndpoint(endpointStr).WithDisableSSL(true).WithCredentials(creds).WithS3ForcePathStyle(true)
+		if len(regionStr) > 0 {
+			cfg = cfg.WithRegion(regionStr)
+		} else {
+			cfg = cfg.WithRegion(DefaultRegion)
+		}
+
 		svc := s3.New(session.New(), cfg)
 		objName := uuid.New().String()
 		fmt.Printf("--- PUT OBJECT %s IN BUCKET %s ---\n", objName, bucketName)
```

Comment:
same here `logrus.Infof()`

Reply:
I have introduced logrus in this package and updated the error logs above. However for this particular print Grant did write it in a particular format using fmt to make it look pretty I think. Also this is the sample app so it is fine I think. I will let Grant take the call if he wants to update this.